src/phoenix/trace/llama_index/callback.py

```diff
@@ -87,6 +89,14 @@ def payload_to_semantic_attributes(
     if event_type in (CBEventType.NODE_PARSING, CBEventType.CHUNKING):
         # TODO(maybe): handle these events
         return attributes
+    if event_type == CBEventType.TEMPLATING:
+        if template := payload.get(EventPayload.TEMPLATE):
+            attributes[LLM_PROMPT_TEMPLATE] = template
+        if template_vars := payload.get(EventPayload.TEMPLATE_VARS):
+            attributes[LLM_PROMPT_TEMPLATE_VARIABLES] = template_vars
+        # TODO(maybe)
+        # EventPayload.SYSTEM_PROMPT
+        # EventPayload.QUERY_WRAPPER_PROMPT
```

Comment:
Keep comments? Not clear what these refer to.

Reply:
these are the extra variables in the payload (see the linked source code), but we don't have a place to store them if they aren't None